scripts/workers/submission_worker.py

```diff
@@ -351,8 +351,16 @@ def run_submission(challenge_id, challenge_phase, submission, user_annotation_fi
                     challenge_phase_split = ChallengePhaseSplit.objects.get(challenge_phase=challenge_phase,
                                                                             dataset_split__codename=split_code_name)
                 except:
-                    stderr.write("ORGINIAL EXCEPTION: No such relation between Challenge Phase and DatasetSplit"
-                                 " specified by Challenge Host \n")
+                    try:
```

Comment:
We don't need a try, catch here. Please refer to the issue for the changes that we need to do. We only need minor changes in the error message.

Reply:
A try-catch-else was added here in so that if no `DatasetSplit` with the given split code name exists, we can handle the case gracefully instead of throwing an exception.

Reply:
Please see that this line https://github.com/Cloud-CV/EvalAI/pull/1785/files?utf8=%E2%9C%93&diff=unified#diff-a428ce55b07affb633407e9bcd4d0aceR351 is already in a try-catch block. So, it will catch if there is an exception.